### PR TITLE
Build ufs_utils on Gaea-C5 (develop branch)

### PR DESCRIPTION
**Pull request by DavidBurrows-NCO**
## DESCRIPTION OF CHANGES: 
Add Gaea-C5 build capability to ufs_utils repository for use with global-workflow.  A module file needs to be updated and standardized and a module setup script needs to be cleaned up.  The module file (modulefiles/build.gaea.intel.lua) is designed after the hera version and the module setup script (sorc/machine-setup.sh) is designed after the ufs-weather-model version.  The required changes have been made and a test build is successful.

## TESTS CONDUCTED: 
Build test complete on Gaea-C5

- [x] Compile branch on all Tier 1 machines using Intel (Orion, Jet, Hera, Hercules and WCOSS2).
- [x] Compile branch on Hera using GNU.
- [x] Run unit tests on Hera.
- [x] Run `link_fixdirs.sh` on all Tier 1 machines.

## DEPENDENCIES:

None.

## DOCUMENTATION:

The main README.md file should be updated.

## ISSUE: 
Fixes issue mentioned in #934 
Contributes to issue mentioned in https://github.com/NOAA-EMC/global-workflow/issues/2535

### CODE REVIEW

File: fix/link_fixdirs.sh
Comment (GeorgeGayno-NOAA) [inline fix/link_fixdirs.sh:1]: Don't forget to update the error messages.

Comment (DavidBurrows-NCO) [reply fix/link_fixdirs.sh:1]: Thanks, George! I updated the error messages for Gaea.
In terms of the testing checklist, do all checklist items need to be completed? I can compile on Jet and Hera. I don't have WCOSS2, Hercules, or Orion access. Do I need to set up the unit tests for Gaea? Thanks for any information.

**Comment by GeorgeGayno-NOAA** [comment]
@DavidBurrows-NCO - since you are not updating source code, I will waive the consistency tests (I will update the PR to reflect this). I can run the unit tests on Hera.

I see there is no way to run the unit tests or consistency tests on Gaea. Since that is not a Tier 1 machine, I won't require it. But I do wonder how you know if UFS_UTILS works on that machine.

Also, 'develop' was updated this morning. Be sure to merge to branch. Thanks.

**Comment by GeorgeGayno-NOAA** [comment]
The README file should be updated: https://github.com/DavidBurrows-NCO/UFS_UTILS/blob/gaea_build/README.md

Add 's4' and 'Gaea' to this line: `On Orion, Hercules, Jet, Hera and WCOSS2 do the following:`

**Comment by DavidBurrows-NCO** [comment]
Good afternoon @GeorgeGayno-NOAA. I updated the README.md file for s4 and gaea.

Our plan with EMC is to get all the submodules built on Gaea (Wei Huang is doing the same for noaaclouds) and merged to the submods, then do a single PR back to global-workflow for Gaea build. We will then begin run time testing at that point.

**Comment by GeorgeGayno-NOAA** [comment]
> Good afternoon @GeorgeGayno-NOAA. I updated the README.md file for s4 and gaea.
> 
> Our plan with EMC is to get all the submodules built on Gaea (Wei Huang is doing the same for noaaclouds) and merged to the submods, then do a single PR back to global-workflow for Gaea build. We will then begin run time testing at that point.

Ok. Let me know when you want to merge this PR.

**Comment by GeorgeGayno-NOAA** [comment]
I updated the required tests under "TESTS CONDUCTED". I can run these tests on WCOSS2, Hercules and Orion for you.

**Comment by DavidBurrows-NCO** [comment]
@GeorgeGayno-NOAA 
I have:
run link_fixdirs.sh on Jet and Hera
I have compiled ufs_utils on Jet and Hera with intel
I have compiled ufs_utils on Hera with gnu
Thanks for running the other tests on machines I don't have access to.
I will move this PR out of draft.
Please let me know if there is anything else.

**Comment by GeorgeGayno-NOAA** [comment]
I successfully tested 4e1ef21 on WCOSS2, Hercules and Orion. It compiled using Intel and the `link_fixdirs.sh` script worked correctly.

**Comment by GeorgeGayno-NOAA** [comment]
All unit tests passed on Hera using 4e1ef21.

**Review by GeorgeGayno-NOAA — APPROVED** [review]

**Comment by DavidBurrows-NCO** [comment]
@GeorgeGayno-NOAA Thanks again for running those additional tests and your help in general!

**Pull request merged**